app/components/ranked-stats/ranked-stats-ctrl.js

```diff
@@ -0,0 +1,67 @@
+angular.module('LeagueViewer')
+  .controller('RankedStatsCtrl', [
+    '$scope',
+    '$location',
+    'lolapi',
+    '$stateParams',
+    '$state',
+    function($scope, $location, lolapi, $stateParams, $state) {
+
+      var summonerName = $stateParams.summonerName;
+      $scope.summonerName = summonerName;
+
+      var onGetSummoner1Error = function(reason) {
+        $scope.errorMessage = 'Could not get Summoner Info';
+      };
+
+      var getSummoner1 = function() {
+        lolapi.getSummoner1(summonerName).then(
+          function(result) {
+            console.log('test');
+            $scope.summonerObject = result[summonerName];
+            summonerId = $scope.summonerObject.id;
+            // getSummonerId(); summId
+            getRankedInfo(summonerId);
+
+          },
+          function(failure) {
+            console.log('fail');
```

Comment:
let's have this call an onError function, similar to what we do in match-history-ctrl